packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -6,115 +6,107 @@ import { View, TouchableHighlight, Text } from 'react-native';
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { useCallback } from '@wordpress/element';
 import { Icon } from '@wordpress/components';
 import { withPreferredColorScheme } from '@wordpress/compose';
 import { __, sprintf } from '@wordpress/i18n';
 import { sparkles } from '@wordpress/icons';
+import { useDispatch } from '@wordpress/data';
+import { store as noticesStore } from '@wordpress/notices';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
 
-class MenuItem extends Component {
```

Comment:
@dnalves what do you think about breaking down the work here into sizable chunks from a commit history standpoint?

Reply:
Great suggestion. I just force pushed with a better commit history.